languages/ert.pot

```diff
@@ -509,10 +509,14 @@ msgid "Create New Event"
 msgstr ""
```

Comment:
Is `languages/ert.pot` meant to be auto generated? @OzzieOrca

Reply:
I think running `yarn angular-gettext-extract` generates it. https://github.com/CruGlobal/conf-registration-web/blob/master/package.json#L76

Do we have any live translations? I don't see a whole lot of tooling for translations. I've added OneSky upload/download scripts for MissionHub web and mobile that run on Travis deploys.

Reply:
ERT isn't officially internationalized, so I don't know why we're doing this anyway.